Steer/DigitizerWorkflow/src/MCHDigitizerSpec.cxx

```diff
@@ -111,15 +110,16 @@ class MCHDPLDigitizerTask
         retrieveHits(mSimChains, "MCHHit", part.sourceID, part.entryID, &hits);
         LOG(INFO) << "For collision " << collID << " eventID " << part.entryID << " found MCH " << hits.size() << " hits ";
```

Comment:
I guess all those LOG(INFO) should be removed (or at least be demoted to LOG(DEBUG)) ?

Reply:
@aphecetche : not clear to me, for the other detectors, I see quite extensive log output. @sawenzel : what do you think?

Reply:
True that we have this in a couple of places, but I guess we can start to reduce this as it was primarily meant for debugging at the beginning.

Reply:
Ok, I replaced now by LOG(DEBUG).